runtime/Makefile.in

```diff
@@ -843,6 +848,12 @@ safe_math_macros.h: safe_math_macros.m4
 safe_math_macros_notmp.h: safe_math_macros_notmp.m4
 	$(M4) < $< > $@
 
+adapter.o: adapter.cpp
+	$(CXX) adapter.cpp -c -o adapter.o
+
+adaptertracking.o: adapter.cpp
+	$(CXX) adapter.cpp -DTRACKING -c -o adaptertracking.o
+
```

Comment:
Makefile.in is auto-generated from Makefile.am. We may not want to change it manually here. Eric is an expert on this subject. @eeide